_posts/2019-05-24-live-view-with-channels.md

```diff
@@ -0,0 +1,499 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: post
+date: 2019-05-24
```

Comment:
do you want to put the current date in?

Reply:
Good catch! I'll update this before we merge :)